libtenzir/builtins/functions/string.cpp

```diff
@@ -140,3 +190,13 @@ TENZIR_REGISTER_PLUGIN(tenzir::plugins::string::trim{"trim", "utf8_trim"})
 TENZIR_REGISTER_PLUGIN(tenzir::plugins::string::trim{"trim_start",
                                                      "utf8_ltrim"})
 TENZIR_REGISTER_PLUGIN(tenzir::plugins::string::trim{"trim_end", "utf8_rtrim"})
+TENZIR_REGISTER_PLUGIN(tenzir::plugins::string::arrow_nullary{
+  "capitalize", "utf8_capitalize"})
+TENZIR_REGISTER_PLUGIN(tenzir::plugins::string::arrow_nullary{"lower",
+                                                              "utf8_lower"})
+TENZIR_REGISTER_PLUGIN(tenzir::plugins::string::arrow_nullary{"reverse",
+                                                              "utf8_reverse"})
```

Comment:
Arrow's `utf8_reverse` operates on code points, not grapheme clusters. This is normally not what you'd want, but I guess it might be good enough for now?

Reply:
I think it's the best we can do for now with reasonable effort.